shared/AppInsightsCore/src/JavaScriptSDK/CookieMgr.ts

```diff
@@ -288,6 +351,24 @@ export function createCookieMgr(rootConfig?: IConfiguration, logger?: IDiagnosti
             if (_isMgrEnabled(cookieMgr)) {
                 // Only remove the cookie if the manager and cookie support has not been disabled
                 result = cookieMgr.purge(name, path);
+            } else {
```

Comment:
The logic of lines 355 to 365 and the formatting of the value are now duplicated here and in the purge function, extract this to a top level function and call from both places to reduce the amount of code